lang/en.json

```diff
@@ -687,7 +689,8 @@
             "metric": "cm",
             "imperial": "in"
         },
-        "you_save": "(You save {amount})",
+        "you_save_1": "(You save",
+        "you_save_2": ")",
```

Comment:
You should change it to a single variable `you_save` like `(You save {price})`. And then you can call it as `{{lang 'products.you_save' price=price.saved.formatted}}`.

Reply:
That unfortunately does not work, and is causing the bug CATALOG-3121. Which is the main reason for the split

Reply:
Can you make that change & show it to me or Michael. I am curious why it does not work.

Reply:
You can see the first "you save" has the accurate value, but the second "you save" one that uses `{{lang 'products.you_save' price=price.saved.formatted}}` does not update the value.
<img width="1679" alt="screen shot 2018-03-26 at 4 43 47 pm" src="https://user-images.githubusercontent.com/37156312/37938929-150f0344-3115-11e8-8168-5e7b4c08be22.png">